sound/soc/sof/pcm.c

```diff
@@ -301,8 +301,12 @@ static int sof_pcm_trigger(struct snd_soc_component *component,
 		ipc_first = true;
 		break;
 	case SNDRV_PCM_TRIGGER_PAUSE_RELEASE:
+		if (pcm_ops && pcm_ops->trigger_ipc_first)
+			ipc_first = true;
```

Comment:
```
		if (pcm_ops && pcm_ops->trigger_ipc_first)
			ipc_first = true;
```
is valid for all cmd, right? How about moving this section outside the switch {} at line 341?